src/stability_sdk/animation.py

```diff
@@ -966,12 +952,15 @@ def transform_video(self, frame_idx) -> Optional[np.ndarray]:
             return mask
         return None
 
-    def _postprocess_inpainting_mask(self, mask, frame_idx, blur_radius=5, min_val=None, max_val=None):
-        mask = cv2.erode(mask, np.ones((blur_radius, blur_radius), np.uint8))
-        mask = cv2.GaussianBlur(mask, (blur_radius*2+1, blur_radius*2+1), 0)
+    def _postprocess_inpainting_mask(
+            self, mask, frame_idx, blur_radius=5, min_val=None, mask_multiplier=None):
+        if mask_multiplier is not None:
+            mask = (mask * mask_multiplier).astype(np.uint8)
+        if not self.args.animation_mode == '3D render':
+            mask = cv2.erode(mask, np.ones((blur_radius, blur_radius), np.uint8))
+            mask = cv2.GaussianBlur(mask, (blur_radius*2+1, blur_radius*2+1), 0)
```

Comment:
Would 3D warp mode benefit from not doing this as well?

Reply:
Made a comparison showing that inpainting model does benefit from mask blurring